### PR TITLE
PSR-494 Updated task list status for disposals

### PR DESCRIPTION
**Pull request by westwater**

### CODE REVIEW

File: app/controllers/nonsipp/TaskListController.scala [hunk 1]
```diff
@@ -456,7 +459,7 @@ object TaskListController {
             .onPageLoad(srn, NormalMode)
```
Comment (kberec) [inline app/controllers/nonsipp/TaskListController.scala:459]: Is the navigation the same for all scenarios?

Comment (westwater) [reply app/controllers/nonsipp/TaskListController.scala:459]: It's not specified as AC but I can route the user straight to the disposal list page when at least 1 disposal is completed - don't know if that would be another story / if we have that on the backlog somewhere or if we do it as part of this

Comment (kberec) [reply app/controllers/nonsipp/TaskListController.scala:459]: I was thinking about the last line of each AC: "and link navigates to...."

Comment (westwater) [reply app/controllers/nonsipp/TaskListController.scala:459]: woops missed that, will change

**Review by kberec — DISMISSED** [review]

**Review by kberec — APPROVED** [review]

**Pull request merged**